src/frontend/src/pages/TeamsPage/ProjectCardsView.tsx

```diff
@@ -0,0 +1,57 @@
+import { Card, CardContent, Typography } from '@mui/material';
+import PageBlock from '../../layouts/PageBlock';
+import { Link as RouterLink } from 'react-router-dom';
+import { Link } from '@mui/material';
+import { ProjectPreview } from 'shared';
+import { routes } from '../../utils/routes';
+import { fullNamePipe, wbsPipe } from '../../utils/pipes';
+import { useSingleProject } from '../../hooks/projects.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { Grid } from '@mui/material';
+
+interface createCardProps {
+  project: ProjectPreview;
+}
+const CreateCard: React.FC<createCardProps> = ({ project }) => {
+  const { isLoading, isError, error, data } = useSingleProject(project.wbsNum);
+  if (isError) return <ErrorPage message={error?.message} />;
+  if (isLoading || !data) return <LoadingIndicator />;
```

Comment:
pretty sure just the isLoading boolean is sufficient. data not existing would either be when loading (covered by isLoading) or when it errors (covered by line 18)

Reply:
The only thing is i need to check somewhere for not data or else it is undefined | null so i would have to force unwrap it when i reference data.

Reply:
peyton is correct here ^ it's a little redundant but it's better than doing `data!` everywhere and costs us nothing